dashboard/app/models/section.rb

```diff
@@ -169,9 +169,12 @@ def summarize
       link_to_assigned = script_path(script)
     end
 
+    teacher = User.find_by id: user_id
+
     {
       id: id,
       name: name,
+      teacherName: teacher.name,
```

Comment:
This could simply be `user.name` or `teacher.name` (removing line 172), I think, taking advantage of [these](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/section.rb#L36-L37) lines.